modules/index/src/test/java/org/apache/ignite/internal/index/IndexManagerTest.java

```diff
@@ -123,8 +123,8 @@ public void setUp() {
     }
 
     @AfterEach
-    void tearDown() throws Exception {
-        IgniteUtils.stopAll(metaStorageManager, catalogManager, indexManager);
+    void tearDown() {
+        assertThat(IgniteUtils.stopAsync(metaStorageManager, catalogManager, indexManager), willCompleteSuccessfully());
```

Comment:
Here and there. Is it possible to use static imports for IgniteUtils or there's clashing?